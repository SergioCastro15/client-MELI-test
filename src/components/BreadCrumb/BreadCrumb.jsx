import React from 'react'
import './BreadCrumb.scss'

export const BreadCrumb = ({ currentCategories }) => (
    <div className="bread-crumb-container">
        {currentCategories.map((category, index) => ((
            <p key={index} className="bread-crumb-info">{category}</p>
        )))}
    </div>
)
