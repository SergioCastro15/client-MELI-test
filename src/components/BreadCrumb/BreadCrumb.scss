@import '../../scss/variables';

.bread-crumb-container {
    display: flex;
    margin: auto;
    flex-direction: row;
    width: 60%;

    .bread-crumb-info {
        margin-right: 11px;
        font-size: 12px;
        color: $gray;

        &:after {
            content: " >";
        }

        &:last-child::after {
            content: "";
        }

        &:hover {
            cursor: pointer;
            color: black;
        }
    }
}

@media (max-width: 768px) {
    .bread-crumb-container {
        width: 90%;
        margin-top: $main-spacing;
    
        .bread-crumb-info {
            margin-top: auto;
            font-size: 7px;
            width: 20%;
            background: #e6e1e2;
            border-radius: 87px;
            height: 30px;

            display: flex;
            align-items: center;
            justify-content: center;


            &:after {
                content: "";
            }
        }
    }
}