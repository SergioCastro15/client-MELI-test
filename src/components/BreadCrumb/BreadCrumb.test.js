import React from 'react'
import { render } from '@testing-library/react'
import { BreadCrumb } from './BreadCrumb'

it('should take a snapshot', () => {
    const currentCategories = ["Audio", "Video", "Musica", "Entretenimiento", "Imagen"]
    const { asFragment } = render(<BreadCrumb currentCategories={currentCategories}/>)
    expect(asFragment(<BreadCrumb />)).toMatchSnapshot()
})