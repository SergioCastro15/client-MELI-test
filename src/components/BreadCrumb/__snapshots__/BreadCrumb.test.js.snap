// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="bread-crumb-container"
  >
    <p
      class="bread-crumb-info"
    >
      Audio
    </p>
    <p
      class="bread-crumb-info"
    >
      Video
    </p>
    <p
      class="bread-crumb-info"
    >
      Musica
    </p>
    <p
      class="bread-crumb-info"
    >
      Entretenimiento
    </p>
    <p
      class="bread-crumb-info"
    >
      Imagen
    </p>
  </div>
</DocumentFragment>
`;
